mods/creative/init.lua

```diff
@@ -54,7 +57,7 @@ function minetest.handle_node_drops(pos, drops, digger)
 	end
 	local inv = digger:get_inventory()
 	if inv then
-		for _, item in ipairs(drops) do
+		for _, item in pairs(drops) do
```

Comment:
pairs is slightly faster than ipairs when dealing with tables not containing numbers.